core/plans/microvm_create_update.go

```diff
@@ -54,40 +53,58 @@ func (p *microvmCreatePlan) Create(ctx context.Context) ([]planner.Procedure, er
 	}
 
 	p.ensureStatus()
-	p.steps = []planner.Procedure{}
+
+	var err error
+	switch {
+	case p.vm.Spec.UpdatedAt != 0:
+		err = p.update(ctx, ports)
+	default:
+		err = p.create(ctx, ports)
+	}
```

Comment:
I think we can remove this completely and just walk through with all the steps, like "create microvm" and the step will say "no it's running, we don't have to do this".

@richardcase: Any idea/plan/suggestion?

Reply:
I had considered this, but I think the mental overhead of keeping them separate isn't very large, and until we actually implement the update logic it's hard to know how different the two are.